apps/backoffice-v2/src/lib/blocks/variants/DefaultBlocks/hooks/useDefaultBlocksLogic/useDefaultBlocksLogic.tsx

```diff
@@ -407,16 +422,27 @@ export const useDefaultBlocksLogic = () => {
     ubosRegistryProvidedBlock,
     websiteBasicRequirementBlock,
     websiteMonitoringBlock,
+    processTrackerBlock,
+    websiteMonitoringBlocks,
+    documentReviewBlocks,
+    businessInformationBlocks,
     workflow?.context?.entity,
   ]);
 
+  const {
+    activeTab,
+    blocks = [],
+    tabs,
+    setTab,
+  } = useCaseBlocksLogic(allBlocks, workflow!.workflowDefinition?.config, { workflow });
```

Comment:
Prefer object syntax at least when there are three or more arguments. Also note `workflow` will be `undefined` frequently so using `!` is risky.
```suggestion
  } = useCaseBlocksLogic({ blocks: allBlocks, config: workflow?.workflowDefinition?.config, workflow });
```